atoMEC/models.py

```diff
@@ -28,19 +28,23 @@ class ISModel:
     """
     The ISModel represents a particular family of AA models known as ion-sphere models.
 
-    The implementation in atoMEC is based on the model described in the following pre-print:
+    The implementation in atoMEC is based on the model described in the pre-print:
+
     T. J. Callow, E. Kraisler, S. B. Hansen, and A. Cangi, (2021).
-    First-principles derivation and properties of density-functional average-atom models. arXiv preprint arXiv:2103.09928.
+    First-principles derivation and properties of density-functional \
+    average-atom models. arXiv preprint arXiv:2103.09928.
 
-    Parameter inputs for this model are related to particular choices of approximation, e.g. boundary conditions
-    or exchange-correlation functional, rather than fundamental physical properties.
+    Parameter inputs for this model are related to particular choices of approximation,\
+    e.g. boundary conditions or exchange-correlation functional, rather than \
```

Comment:
```suggestion
    Parameter inputs for this model are related to particular choices of approximation,
    e.g. boundary conditions or exchange-correlation functional, rather than
```